### PR TITLE
update synth, inst and sim tex files processing

### PR DESCRIPTION
**Pull request by codingUniv**

### CODE REVIEW

File: build/doc/tsrc/ug.tex [hunk 1]
```diff
@@ -94,11 +94,11 @@ \subsection{Software Accessible Registers}
 
 \section{Usage}
 \label{sec:inst}
-\input{inst_lib}
+\input{inst}
```
Comment (jjts) [inline build/doc/tsrc/ug.tex:97]: Why inst an not instance?

Files need different names if they have  different functions. 

The names should reflect the function. Inst and instance have different names for no reason.  What do they do differently  ?

They should have same names if the local should replace the lib one.

They should have different names, for example, instance and instance_description, if the latter is included I  the former and provides a description.

**Review by jjts — COMMENTED** [review]
Unexplained

File: build/doc/tsrc/ug.tex (hunk 1 above)
Comment (codingUniv) [reply build/doc/tsrc/ug.tex:97]: iob-lib's synth, sim and inst tex files are not replaced by IP cores' local files but they include them.
In this PR _suffix is also removed from inst and sim lib's tex files.

**Comment by codingUniv** [comment]
Please refer to the new commit.

**Pull request merged**